vizro-core/docs/pages/user-guides/titles-header-footer.md

```diff
@@ -0,0 +1,174 @@
+# How to add titles, headers and footers
+
+This guide explains how to add titles, headers, and footers to your components in Vizro.
+
+The [`Graph`][vizro.models.Graph], the [`Table`][vizro.models.Table] and the [`AgGrid`][vizro.models.AgGrid] components
+accept a `title`, `header` and `footer` argument. This is useful for providing context to the data being
+displayed, or for adding a description of the data.
+
+- **title**: Displayed as an H3 header, useful for summarizing the main topic or insight of the component.
+- **header**: Accepts Markdown text, ideal for additional descriptions, subtitles, or detailed data insights.
+- **footer**: Also accepts Markdown text, commonly used for citing data sources, providing information on the last update, or adding disclaimers.
+
+
+!!! note
+
+    Although you can directly provide a `title` to the Plotly Express chart, we recommend using `Graph.title` for
+    proper alignment with other components on the screen.
+
+## Formatted Graph
+
+!!! example "Formatted Graph"
+    === "app.py"
+        ```{.python pycafe-link}
+        import vizro.models as vm
+        import vizro.plotly.express as px
+        from vizro import Vizro
+
+        df = px.data.iris()
+
+        page = vm.Page(
+            title="Formatted Graph",
+            components=[
+                vm.Graph(
+                    figure=px.scatter(iris, x="sepal_width", y="sepal_length", color="species"),
```

Comment:
```suggestion
                    figure=px.scatter(df, x="sepal_width", y="sepal_length", color="species"),
```
Discovered actually with PyCafe :). The example still does not run, but will once we release this change.

That means that this specific cell won't run in the `latest` docs, but it will run after release. Do you think that's fine?

Reply:
Yes, I think that's fine 👍 Not sure how we would do it otherwise. Adding the examples in after the release doesn't really make sense, because then the feature is not really "ready".